controllers/construct/mongodbstatefulset.go

```diff
@@ -64,7 +64,7 @@ const (
 	automationAgentLogOptions = " -logFile ${AGENT_LOG_FILE} -maxLogFileDurationHrs ${AGENT_MAX_LOG_FILE_DURATION_HOURS} -logLevel ${AGENT_LOG_LEVEL}"
 
 	MongodbUserCommand = `current_uid=$(id -u)
-AGENT_API_KEY="$(cat /mongodb-automation/agent-api-key/agentApiKey)"
+AGENT_API_KEY="$(cat /mongodb-automation/agent-api-key/agentApiKey 2>/dev/null)"
```

Comment:
We reference this const for AppDB construction as well. What's the interaction there?